app/src/main/java/be/huffle/allowanceplanner/activities/AddAllowanceActivity.java

```diff
@@ -39,42 +40,42 @@ protected void onCreate(Bundle savedInstanceState)
 		actionBar.setDisplayHomeAsUpEnabled(true);
 		actionBar.setTitle("Add Allowance");
 
-		addButton.setOnClickListener(this);
+		//addButton.setOnClickListener(this);
 	}
 
 	@Override
 	public boolean onOptionsItemSelected(@NonNull MenuItem item)
 	{
-		int id = item.getItemId();
-
-		if (id == android.R.id.home)
+		switch (item.getItemId())
 		{
-			finish();
-		}
-
-		return true;
-	}
-
-	@Override
-	public void onClick(View view)
-	{
-		switch (view.getId())
-		{
-			case R.id.add_button:
+			case android.R.id.home:
```

Comment:
```suggestion
			case R.id.home:
```

Reply:
_android_ needs to stay for the back button to work